vme/src/utility.cpp

```diff
@@ -76,146 +72,64 @@ int dice(int number, int size)
 
     for (r = 1; r <= number; r++)
     {
-#ifdef GENERIC_SYSV
```

Comment:
Pretty sure we don't need system 5 unix - because the brace appears outside the #def it messes up the indent - so just got rid of the #def